.github/workflows/sitemap-generator.yml

```diff
@@ -25,5 +25,6 @@ jobs:
       - uses: fregante/setup-git-user@v1
       - name: Commit & push sitemap
         run: |-
+          git add sitemap.xml
```

Comment:
The `-a` in the commit command does this already, but sure

Reply:
only already staged files I think. because it failed saying that sitemap.xml could not be added